www-servers/pound/pound-4.16.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# TODO: Add adns as a dependency in order to allow building with support for
+#       dynamic backends
+# TODO: Add hoard as a dependency in order to support the --enable-hoard configure
+#       argument
+
+DESCRIPTION="A http/https reverse-proxy and load-balancer"
+HOMEPAGE="https://github.com/graygnuorg/pound"
+SRC_URI="https://github.com/graygnuorg/pound/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="tcmalloc test"
+
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-libs/libpcre2:=
+	dev-libs/openssl:=
+	tcmalloc? ( dev-util/google-perftools )
+	test? (
```

Comment:
Let's put these test deps only in DEPEND (or BDEPEND is fine, as we don't do tests for cross).